.travis.yml

```diff
@@ -18,9 +18,20 @@ before_install:
             sudo docker pull welder/web:latest
             sudo docker pull welder/web-with-coverage:latest
             sudo docker pull welder/web-e2e-tests:latest
-        else
```

Comment:
`npm install` inside the else block, we don't need a second if statement with a negative condition to do that.

Reply:
updated.